packages/vue-client/tsconfig.config.json

```diff
@@ -3,6 +3,14 @@
   "include": ["vite.config.*", "vitest.config.*", "cypress.config.*"],
```

Comment:
I'm pretty sure the changes were required in all 3 tsconfig files (excluding tsconfig.vitest.json). Maybe there is a way to dedupe using the "extends" field.  But I don't quite understand the difference in intent behind `app` and `config` anyway...  Maybe we can remove them and just keep tsconfig.json?